<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reservation2</class>
 <widget class="QWidget" name="reservation2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1192</width>
    <height>791</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="f_liste_courses">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>70</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI Semilight</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Listes Reservation</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>1121</width>
     <height>521</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="border1">
    <widget class="QLabel" name="course_name">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Nirmala UI Semilight</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="course_type">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>160</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Nirmala UI Semilight</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>ID Réservation</string>
     </property>
    </widget>
    <widget class="QLabel" name="course_beginning">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>260</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Nirmala UI Semilight</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Enrolement date</string>
     </property>
    </widget>
    <widget class="QLabel" name="course_ending">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>70</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Nirmala UI Semilight</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Ending date</string>
     </property>
    </widget>
    <widget class="QLabel" name="course_description">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>160</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Nirmala UI Semilight</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Description</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="put_name">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>80</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Name</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="put_description">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>170</y>
       <width>211</width>
       <height>121</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Bio</string>
     </property>
    </widget>
    <widget class="QPushButton" name="confirm_add">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>350</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Confirm add</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="confirm_update">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>350</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Confirm update</string>
     </property>
    </widget>
    <widget class="QLabel" name="confirm_icon">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>320</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="res.qrc">:/icon/check-mark_1207330.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="update_icon">
     <property name="geometry">
      <rect>
       <x>980</x>
       <y>330</y>
       <width>35</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="res.qrc">:/icon/Untitled.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="dateTimeEdit">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>260</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QDateTimeEdit {
    font: &quot;Poppins&quot;;
    border: 2px solid #3498db;
    border-radius: 5px;
    padding: 3px;
    background: #f8f8f8;
}

QDateTimeEdit::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: center right; /* Ajusté pour centrer verticalement */
    width: 15px;
    border-left: 2px solid #3498db;
    margin-right: 5px;
}

QDateTimeEdit::down-arrow {

    width: 10%;
    height: 10%;
    margin-right: 5px;
}

QDateTimeEdit::up-button, QDateTimeEdit::down-button {
    width: 10%; 
    height: 10%;
    image: url(&quot;C:/Users/WorKenX/Desktop/Project qt/i/icon/fleche2.png&quot;); 
    margin-right: 5px;
}

QDateTimeEdit::down-button {
    image: url(&quot;C:/Users/WorKenX/Desktop/Project qt/i/icon/fleche.png&quot;);
    margin-right: 5px;
}</string>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="dateTimeEdit_2">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>80</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QDateTimeEdit {
    font: &quot;Poppins&quot;;
    border: 2px solid #3498db;
    border-radius: 5px;
    padding: 3px;
    background: #f8f8f8;
}

QDateTimeEdit::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: center right; /* Ajusté pour centrer verticalement */
    width: 15px;
    border-left: 2px solid #3498db;
    margin-right: 5px;
}

QDateTimeEdit::down-arrow {

    width: 10%;
    height: 10%;
    margin-right: 5px;
}

QDateTimeEdit::up-button, QDateTimeEdit::down-button {
    width: 10%; 
    height: 10%;
    image: url(&quot;C:/Users/WorKenX/Desktop/Project qt/i/icon/fleche2.png&quot;); 
    margin-right: 5px;
}

QDateTimeEdit::down-button {
    image: url(&quot;C:/Users/WorKenX/Desktop/Project qt/i/icon/fleche.png&quot;);
    margin-right: 5px;
}</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ID">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>170</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Name</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="border2">
    <widget class="QLineEdit" name="search_course">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QPushButton" name="search_button">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>30</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/icon/search_5807787.png</normaloff>:/icon/search_5807787.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="course_name_7">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>30</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Nirmala UI Semilight</family>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 127);</string>
     </property>
     <property name="text">
      <string>Sort by</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="sort_by">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>30</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Type formation</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sort_button">
     <property name="geometry">
      <rect>
       <x>950</x>
       <y>30</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/icon/setting_6874449.png</normaloff>:/icon/setting_6874449.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="course_name_8">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>430</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Nirmala UI Semilight</family>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 127);</string>
     </property>
     <property name="text">
      <string>Generate pdf</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="generate_pdf">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>420</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/icon/pdf_8304568.png</normaloff>:/icon/pdf_8304568.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="course_name_9">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>420</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Nirmala UI Semilight</family>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 127);</string>
     </property>
     <property name="text">
      <string>Generate certif</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="generate_certif">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>420</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/icon/diploma_4706145.png</normaloff>:/icon/diploma_4706145.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="course_name_10">
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>390</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Nirmala UI Semilight</family>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 127);</string>
     </property>
     <property name="text">
      <string>Update</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="update_button">
     <property name="geometry">
      <rect>
       <x>970</x>
       <y>390</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/icon/edit_1443396.png</normaloff>:/icon/edit_1443396.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="course_name_11">
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>450</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Nirmala UI Semilight</family>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 127);</string>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="delete_2">
     <property name="geometry">
      <rect>
       <x>970</x>
       <y>450</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/icon/delete_3983957.png</normaloff>:/icon/delete_3983957.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableau_historique">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>100</y>
       <width>511</width>
       <height>261</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Description</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
      <property name="background">
       <color>
        <red>100</red>
        <green>161</green>
        <blue>194</blue>
       </color>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Enrolement_date</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Ending_date</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="border3">
    <widget class="QLabel" name="stat1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>511</width>
       <height>261</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="res.qrc">:/icon/f.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="stat2">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>80</y>
       <width>531</width>
       <height>261</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="res.qrc">:/icon/e.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>80</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/icon/statii.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/icon/plus2.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="f_add_course">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI Semilight</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Add reservation</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="f_statistics">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>70</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI Semilight</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Statistics</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>80</y>
     <width>30</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/icon/list1.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>f_statistics</zorder>
  <zorder>f_add_course</zorder>
  <zorder>f_liste_courses</zorder>
  <zorder>stackedWidget</zorder>
  <zorder>label_3</zorder>
  <zorder>label_2</zorder>
  <zorder>label</zorder>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
